patient-view-be/src/main/java/spvproject/patientviewbe/Controller/AnamnesisController.java

```diff
@@ -0,0 +1,16 @@
+package spvproject.patientviewbe.Controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import spvproject.patientviewbe.Model.AnamnesisModel;
+
+@RestController
+@RequestMapping("/anamnesis")
+public class AnamnesisController {
+
+    @GetMapping("/first")
```

Comment:
What does `first` mean? Routes should have meaning and be intuitive. In this simple case I would recommend having this route be called `data`, as you are returning anamnesis data.
The request will then look like `GET /anamnesis/data`